### DISCUSSION TITLE
Does vscode-neovim have command-line history list feature?

### DISCUSSION THREAD

**Discussion by dhythm** [vscode-neovim/vscode-neovim#1108] [Q&A]
I started using this extension. It very helps to code with vim's feature.
Now, I'm looking for the feature of command-line history list like the one below.

![Screen Shot 2022-12-20 at 0 31 16](https://user-images.githubusercontent.com/10891318/208462864-28ba3ef0-e0a0-40ab-bb64-dc674d4d6c73.png)

![Screen Shot 2022-12-20 at 0 31 36](https://user-images.githubusercontent.com/10891318/208462909-c4766a18-a810-4cf0-bfae-e84ae1aca617.png)
e.g.) VScodeVim shows emulated list by using vscode command list.

I looked and searched in issues, but most of the issues discussed how to navigate prev / next history not about the list of history.

Does this extension have the feature of a command line history list?

**Comment by theol0403** [comment]
https://github.com/vscode-neovim/vscode-neovim/issues/294

**Reply by dhythm** [reply]
Sorry for late reply. Thanks for your comment 🙏

**Comment by theol0403** [comment]
Two options:
- navigate history in cmdline using c-p
- find a way to print history to the output without entering ex command-line
